# Minimal protein list for chromatin replication-coupled symmetric histone recycling revealed through in vitro reconstitution
Fritz Nagae, Shizuko Endo, Yasuto Murayama, Tsuyoshi Terakawa

## Abstract
Upon eukaryotic DNA replication, symmetric histone recycling from parental to daughter strands is vital for transmitting epigenetic information to the next generations. Recent genome-wide sequencing studies have identified several protein regions contributing to symmetric histone recycling in a cell. However, a minimal list of proteins for symmetric histone recycling remains unknown. Here, we successfully reconstituted histone recycling with ~30 purified proteins and analyzed the products digested by Micrococcal nuclease with the newly developed pipeline called a Repli-pore-seq in which nanopore sequencing and deep-learning-based classification were combined. As a result, we confirmed that the histones were recycled symmetrically to the lagging and leading strands. The recycled histones form tetrasomes or hexasomes and are deposited on the DNA sequences energetically favorable for forming nucleosomes. We also observed the discordance of the recycled position between lagging and leading strands on the GC-rich DNA sequences. Among proteins dispensable for chromatin replication, the lagging-strand maturation factors, Fen1/Cdc9, were also dispensable for symmetric recycling. The removal of Pol Î´ disrupted the recycling symmetry, and that of Ctf4 or Csm3/Tof1 altered the recycling position. These findings provide critical insights into the molecular players and mechanisms underlying symmetric histone recycling.

## Contents
This repository contains PDF file for the manuscript, the supplementary data, and the codes

### pARS1-4.1kb.fasta
The DNA sequence for pARS1-4.1kb

### repli-pore-seq-classifier
The codes to classify biotinylated DNA in the Repli-pore-seq pipeline. The latest version may be available at https://github.com/Fritz1414213562/repli-pore-seq-classifier.
